components/ui/ProjectCard.vue

```diff
@@ -40,7 +41,7 @@
                   >Draft</span
                 >
                 <span v-if="status === 'processing'" class="badge yellow">
-                  Processing
+                  Processing/Under Review
```

Comment:
An `or` is more visually appealing. Additionally, try not to use Pascal Case. It is out of style, plus just a tiny bit harmful to accessibility if used too much

Reply:
I can use an `or`, it just makes the already large badge larger. I like having `Review` capitalized, but I'll change it.

Reply:
Neither `or` nor Sentence case look more visually appealing to me after trying them.

Which of the options below do you like the most? Or did you have something else in mind?

\#&#x2060;1
![image](https://user-images.githubusercontent.com/44736536/118195615-63112600-b400-11eb-83d8-80044fdd5bcd.png)
\#&#x2060;2
![image](https://user-images.githubusercontent.com/44736536/118196060-34e01600-b401-11eb-9684-515112558c3b.png)
\#&#x2060;3
![image](https://user-images.githubusercontent.com/44736536/118195674-84721200-b400-11eb-8699-b97f27486716.png)
\#&#x2060;4
![image](https://user-images.githubusercontent.com/44736536/118195727-9a7fd280-b400-11eb-964f-edcc580e411e.png)
\#&#x2060;5
![image](https://user-images.githubusercontent.com/44736536/118195785-b97e6480-b400-11eb-8bb9-2c22b5d4e711.png)

Personally, I like #&#x2060;6. #&#x2060;3 is okay too.

Reply:
Personally, I'd just prefer `Under review`, no need to even have `Processing` there imo, but false may have something else in mind

Reply:
If I remember correctly, the Processing was for when Modrinth checked files automatically, but seeing that doesn't exist yet, won't be a thing for a while, and should probably have it's own status, I agree it should just be `Under review`

Reply:
I agree, `Under review` makes the most sense currently. If/when the backend starts processing files, this will be easy to change.